app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.where(author_id: params[:user_id])
+    @users = User.find(params[:user_id])
+  end
+
+  def new
+    @post_new = Post.new
+    @user_new = current_user
+    @like = Like.new
+  end
```

Comment:
- [ ] when i try to access the route user/:userid/posts/new I have this error , this is because you do not have the method current_user in your application controller. kindly fix that
![Screenshot from 2023-06-23 20-58-23](https://github.com/SabaAhmad404/Blog-App-Rails/assets/66097371/1a374bad-b3f7-4d46-9612-9a2781aeafc9)